lib/membrane/rtp/session/report.ex

```diff
@@ -48,11 +38,22 @@ defmodule Membrane.RTP.Session.Report do
     report_data = %{
       report_data
       | remote_ssrcs: remote_ssrcs,
-        remote_reports: remote_reports,
-        stats: []
+        remote_reports: remote_reports
     }
 
-    {maybe_report, remote_ssrcs, report_data}
+    {remote_ssrcs, report_data}
+  end
+
+  @spec ensure_nothing_to_report(Data.t()) :: {maybe_report_t, Data.t()}
+  def ensure_nothing_to_report(report_data) do
```

Comment:
Still no timeout, but I can live with that. However, I'd change the name to `flush_report`